code/game/objects/items/weapons/storage/stickersheets.dm

```diff
@@ -16,6 +16,22 @@
 		/obj/item/sticker
 	)
 
+// List/Procs to list all stickers for custom stickersheet(similar to lunchbox code)
+var/list/stickersheet_stickers_ = typesof(/obj/item/sticker)
+
+/proc/stickersheet_stickers()
```

Comment:
DMDoc and why is this a global proc instead of being under /datum/gear/toy/stickersheet_custom?